src/test/java/com/wikia/webdriver/testcases/mercurytests/RedirectionTests.java

```diff
@@ -36,4 +43,52 @@ public void RedirectionTest_001_RedirectFromWWW() {
     new ArticlePageObject(driver).openWikiPage(url);
     Assertion.assertUrlEqualToCurrentUrl(driver, expectedUrl);
   }
+
+  @Test(groups = "RedirectionTest_002")
+     @Execute(onWikia = "mercuryautomationtesting")
+     public void RedirectionTest_002_RedirectFromFullSiteToOasis() {
+    MercuryFooterComponentObject footer = new MercuryFooterComponentObject(driver);
+    com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject article;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "article" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)